logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -1,7 +1,9 @@
 package com.wire.kalium.logic.data.message
 
-// IDE is currently complaining that it's not implemented on JVM. The compiler builds it correctly. The IDE is wrong.
-expect class ProtoContentMapper {
+interface ProtoContentMapper {
     fun encodeToProtobuf(protoContent: ProtoContent): PlainMessageBlob
     fun decodeFromProtobuf(encodedContent: PlainMessageBlob): ProtoContent
 }
+
+// IDE is currently complaining that it's not implemented on JVM. The compiler builds it correctly. The IDE is wrong.
+expect class PlatformProtoContentMapper: ProtoContentMapper
```

Comment:
Why the `Platform`prefix? Wouldn't it be more consistent with `ProtoContentMapperImpl`?